docs-zh/source/faq/build.md

```diff
@@ -23,18 +23,32 @@
            JAVA_LDFLAGS="$JAVA_LDFLAGS -lzstd"
        fi
    ```
-  
+
 
 ## rocksdb编译问题
 
 编译纠删码子系统显示报错 `fatal error: rocksdb/c.h: no such file or directory...`
-- 首先确认`.deps/include/rocksdb`目录下是否存在报错所指向的文件， 
+- 首先确认`.deps/include/rocksdb`目录下是否存在报错所指向的文件，
 - 如果存在 可`source env.sh`后再次尝试，如果没有该文件或者仍然报错，可将`.deps`目录下rocksdb相关的文件全部清理，然后重新编译。
-   
+
 ## cannot find -lbz2
 
 编译时候如果报错 `/usr/bin/ld: cannot find -lbz2`，确认是否安装`bzip2-devel`（版本1.0.6及以上）
 
 ## cannot find -lz
 
-编译时候如果报错 `/usr/bin/ld: cannot find -lz`，确认是否安装`zlib-devel`（版本1.2.7及以上）
\ No newline at end of file
+编译时候如果报错 `/usr/bin/ld: cannot find -lz`，确认是否安装`zlib-devel`（版本1.2.7及以上）
+
+## `cc1plus: all warnings being treated as errors` 构建`rocksdb`时报错
+
+- 对于`blobstore`模块来说，进入目录`blobstore`后，在`blobstore/env.sh`文件中添加以下新的环境变量后执行`source env.sh`后继续编译:
+
+```bash
+export DISABLE_WARNING_AS_ERROR=true
+```
+
+- 对于`cubefs`根目录而言，请在根目录下的`env.sh`中添加导出环境变量：
+
+```bash
+export CXXFLAGS=-Wno-error
```

Comment:
hi,cloorc, I am not sure the flag wno-error could be supported in different gcc version, as https://gcc.gnu.org/bugzilla/show_bug.cgi?id=65403

Reply:
Hmm, it's might be version dependent. What if adding a specific gcc version as extra condition? Of course, flags here are optional. Develoeprs might add whatever options accordingly. And it works in my circumstance with specific gcc version.

Reply:
> Hmm, it's might be version dependent. What if adding a specific gcc version as extra condition? Of course, flags here are optional. Develoeprs might add whatever options accordingly. And it works in my circumstance with specific gcc version.

I consider "flags here are optional" should make a declaration. or else users may make a mistake

Reply:
Ok, I've updated related description, please have a look again. Thanks!